src/api/utils/initial_data.py

```diff
@@ -47,48 +49,106 @@
                                  description="Mentoring students outside of"
                                  " Andela e.g. via SheLovesCode",
                                  value=250)
+activity_types = [
+    interview, open_saturdays, tech_event, open_source, hackathon, blog,
+    app, mentor, marketing, press, outside_mentoring
+]
 
 # societies
 phoenix = Society(name="Phoenix")
 istelle = Society(name="iStelle")
 sparks = Society(name="Sparks")
 invictus = Society(name="Invictus")
+societies = [phoenix, istelle, sparks, invictus]
 
 # test countries
 kenya = Country(name='Kenya')
 
-# test user
-user = User(
-    uuid="-KdQsMt2U0ixIy_-yJEH",
+# cohorts
+cohort_14_ke = Cohort(name='Cohort 14', country=kenya)
+
+## users
+# member
+member = User(
+    uuid="-KdQsMtixI2U0y_-yJEH",
     name="Test User",
     photo="https://lh6.googleusercontent.com/-1DhBLOJentg/AAAAAAAAA"
-          "AI/AAAAAAAAABc/ImM13eP_cAI/photo.jpg?sz=50",
-    email="lawrence.wachira@andela.com",
+          "AI/AAAAAAAAABc/ImeP_cAI/photo.jpg?sz=50",
+    email="test.user.societies@andela.com",
+    country=kenya,
+    cohort=cohort_14_ke,
+    society=phoenix
+)
+
+# president
+president = User(
+    uuid="-KdQsMtixG4U0y_-yJEH",
+    name="Test President",
+    photo="https://lh6.googleusercontent.com/-1DhBLOJentg/AAAAAAAAA"
+          "AI/AAAAAAnAABc/ImeP_cAI/photo.jpg?sz=50",
+    email="test.president.societies@andela.com",
+    role="president",
```

Comment:
The current implementation of checking the user roles is through the payload with the decorator factory.
Adding this column in the DB I find unnecessary.

Reply:
I'm using the models as they are on develop @Andretalik. With a test for seeding up now, changing that to something different would fail given how the models are defined. Also why I told you to take the code in this file, its test, update it on your branch to match your models' changes and `test it`.